### PR TITLE
Sort apps by stars, remove installations as the stats are not reliable

### PR DESCRIPTION
**Pull request by AaronDewes**
This fixes the app sorting by sorting by installations, then stars

### CODE REVIEW

**Comment by AaronDewes** [comment]
@gr2m Unrelated, but what do you think about adding depndabot to this repo and the main probot repo?

**Comment by gr2m** [comment]
> what do you think about adding depndabot to this repo and the main probot repo

I think we have dependabot setup. But after using it for a while here and other projects, I migrated to @renovateapp and I'd like to do the same with Probot. 

It's as good a time as any to do that, I even wrote a script to make that setup quick and easy for an org:
https://github.com/octoherd/script-setup-renovate/

Do you have any objections against renovate or preference for Dependabot? I found Dependabot is not working well for libraries, it is made for end-user applications.

**Review by gr2m — COMMENTED** [review]
I think the number of installations is no longer accurate. We cannot load it without the apps exposing it, and the latest probot no longer does. I wish the number of installs was public information but unfortunately it's not, so maybe instead of showing a number that nolonger represents the current state, maybe we should remove it altogether?

For example, the WIP app has currently 9177 installations, I publish some stats to https://wip.vercel.app/stats. But the probot pages says 1366

I can ask GitHub to maybe somehow give us these numbers manually, or we can ask folks to setup a `stats` action similar to what I did here: https://github.com/wip/app/actions/runs/1002788503. We could create this stats action and have it report the install numbers to us? 

Either way, I'll create a follow up issue, but for now I'd say let's remove the installation count? What do you think?

**Comment by AaronDewes** [comment]
> I can ask GitHub to maybe somehow give us these numbers manually

Please do, that would also be interesting to see anyway for all apps.

> or we can ask folks to setup a stats action similar to what I did here: https://github.com/wip/app/actions/runs/1002788503. We could create this stats action and have it report the install numbers to us?

That would be awesome too, it could also be in `crete-probot-app` by default.

> Either way, I'll create a follow up issue, but for now I'd say let's remove the installation count? What do you think?

Agree, I'll change this then.

**Comment by AaronDewes** [comment]
> Do you have any objections against renovate or preference for Dependabot? I found Dependabot is not working well for libraries, it is made for end-user applications.

No, Dependabot is just to easiest to configure in my opinion and I am already familiar with it, I didn't know about renovate

**Comment by gr2m** [comment]
shall we remove the install counts in a separate PR or do you want to add it to this one?

**Comment by AaronDewes** [comment]
I'll remove it here

**Comment by AaronDewes** [comment]
Should the other data from the stats API (popular orgs using this) be removed too?

**Comment by gr2m** [comment]
> popular orgs using this

I think that information at least is public, at least on the marketplace? e.g. https://github.com/marketplace/wip 
image

Let's revisit that later ok?

**Comment by AaronDewes** [comment]
> I think that information at least is public, at least on the marketplace? e.g. https://github.com/marketplace/wip

Oh, I didn't see that. Still, what should I do now? Remove that information from the website or keep it?

**Comment by gr2m** [comment]
I'd say keep it for now

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -28,6 +28,8 @@ jobs:
           yarn
           yarn sass
           yarn build
+          # Prevent GitHub from trying to use Jekyll
+          touh _site/.nojekyll
```
Comment (gr2m) [inline .github/workflows/build.yml:32]: was this meant to be `touch`?

```suggestion
          touch _site/.nojekyll
```

File: script/sync.ts [hunk 2]
```diff
@@ -76,16 +76,19 @@ files.forEach(async (path) => {
     //@ts-expect-error
     app.updated = new Date(repo.data.pushed_at).toRubyString();
 
+    // app.installations is no longer used
+    if (app.installations) delete app.installations;
```
Comment (gr2m) [inline script/sync.ts:80]: shall we removed this as you removed it from the source files?

**Review by gr2m — APPROVED** [review]
great work, thanks Aaron!

**Comment by robvanderleek** [comment]
Maybe I missed something in the discussion above but the current app list on https://probot.github.io/apps/ seems to be sorted in a reverse alphabetically way, not by the number of stars. Should I open a new issue to have it sorted by stars?

**Pull request merged**